tests/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Tests from flow_manager napp."""
+
+import sys
+import os
+from pathlib import Path
+
+if 'VIRTUAL_ENV' in os.environ:
+    BASE_ENV = Path(os.environ['VIRTUAL_ENV'])
+else:
+    BASE_ENV = Path('/')
+
+NAPPS_PATH = BASE_ENV / '/var/lib/kytos/napps/kytos/'
```

Comment:
If this is really necessary, the right path should be `BASE_ENV / '/var/lib/kytos/'`, because the imports should be `napps.kytos.flow_manager`, resolving to `napps/kytos/flow_manager`.